Corgibytes.Freshli.Cli/Functionality/Git/GitManager.cs

```diff
@@ -6,14 +7,24 @@ public class GitManager : IGitManager
 {
     [JsonProperty] private readonly GitArchive _gitArchive;
 
-    public GitManager(GitArchive gitArchive)
-    {
-        _gitArchive = gitArchive;
-    }
+    public GitManager(GitArchive gitArchive) => _gitArchive = gitArchive;
 
     public string CreateArchive(
         string repositoryId, GitCommitIdentifier gitCommitIdentifier) =>
         _gitArchive.CreateArchive(repositoryId, gitCommitIdentifier);
 
+    public bool GitRepositoryInitialized(string repositoryLocation, IConfiguration configuration)
```

Comment:
I think `IConfiguration` is better injected via the constructor then forcing the caller to know how to pass it in.